e2e/regression_guardrail/guardrail.insulin_sensitivities.spec.js

```diff
@@ -0,0 +1,65 @@
+const { loop } = require('../../src/index');
+
+describe('guardrail insulin sensitivities', () => {
+    it('cannot set 501 units', async () => {
+        await loop.Launch();
+        await loop.Configure({
+            settings: {
+                AddPumpSimulator: true,
+                InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '501' }],
+            }
+        });
+        //TODO assert cannot be set
+    });
+    it('set 500 units, warning', async () => {
+        await loop.Launch();
+        await loop.Configure({
+            settings: {
+                AddPumpSimulator: true,
+                InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '500' }],
+            }
+        });
+        //TODO assert on warning
+    });
+    it('set 400 units, warning', async () => {
+        await loop.Launch();
+        await loop.Configure({
+            settings: {
+                AddPumpSimulator: true,
+                InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '400' }],
+            }
+        });
+        //TODO assert on warning
+    });
+    it('set 15 units, warning', async () => {
```

Comment:
[nit] I would only have a pair of tests after every "boundary". So, 501 (to show it cannot be set) and 500 (to show that it can be set with a warning). Then, if one type of alert is given for 400+, check 400 and 399. If another type of alert is given for 200+, check 200 and 199. Etc.. A "boundary" is where the fundamental behavior changes. Does that make sense?